src/pages/en/developers/api-registration/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Developer portal and registration
+order: -12
+---
+
+## Digitransit developer portal
+Starting from 3.4.2023, the use of the Digitransit production APIs ([api.digitransit.fi](https://api.digitransit.fi), [cdn.digitransit.fi](https://cdn.digitransit.fi) and [digitransit-prod-cdn-origin.azureedge.net](https://digitransit-prod-cdn-origin.azureedge.net)) will require registration and use of API keys. Registration can be done at the [Digitransit developer portal](https://portal-api.digitransit.fi/). After the registration, it's possible to create subscriptions to a product and the API keys related to the subscriptions will also be available at the developer portal.
+
+## Transition period
+Developers will have until 3.4.2023 to register at [the developer portal](https://portal-api.digitransit.fi/) and to start using API keys in the requests. After that, we will prevent the anonymous use of our production APIs.
+
+## Use of API keys
+The API keys can be fetched from the developer portal's profile page. Each subscription contains two keys so it is possible to regenerate the second key while the other key is in use, if there is a need. The API key can be included either as a URL parameter or as a header. The parameter and the header name are both `digitransit-subscription-key` and the value should be the key.
```

Comment:
API keys can be fetched from the developer portal's profile page. Each subscription contains two keys, so that it is possible to regenerate a second key while the other key is in use, if there is a need. An API key can be included either as a URL parameter or as a header. The parameter and the header name are both `digitransit-subscription-key` and the value should be the key.